src/components/common/ColonyHome/ColonyHomeLayout.tsx

```diff
@@ -71,7 +69,10 @@ const ColonyHomeLayout = ({
                   />
                 </div>
                 {showActions && (
-                  <NewActionButton /* ethDomainId={ethDomainId} */ />
+                  <NewActionButton
+                    colony={colony}
```

Comment:
We're trying to go away from passing the current colony as a prop and tend to get it from the ColonyContext in the component itself:
```js
const { colony } = useColonyContext();
```